.setup/install_system.sh

```diff
@@ -536,7 +536,7 @@ EOF
     # being that we do not disable phpinfo() on the vagrant machine as it's not a function that could be used for
     # development of some feature, but it is useful for seeing information that could help debug something going wrong
     # with our version of PHP.
-    DISABLED_FUNCTIONS="popen,pclose,proc_open,chmod,php_real_logo_guid,php_egg_logo_guid,php_ini_scanned_files,"
```

Comment:
chmod was intentionally on our list of disabled functions:
https://submitty.org/sysadmin/installation/system_customization#disable-php-functions

If we can find an alternate workaround that would be preferred.

Reply:
Twig calls it and causes an issue if we don't let it. There isn't a way around that unless we modify their source code. I'm of course not changing the version of Twig so the code isn't changing. It seems that PHP 7 was happy ignoring errors of a disabled function being called. PHP 8 is more strict and now errors in that case. While I don't like removing functions from this list, I don't think removing chmod is of high issue/concern.

Reply:
ok, we'll have to update the submitty.org docs too.